### DISCUSSION TITLE
Scoop not working for Rancher Desktop - cannot find manifest file

### DISCUSSION THREAD

**Discussion by ghost** [rancher-sandbox/rancher-desktop#4950] [Q&A]
Hi, 

When installing rancher desktop using the scoop command - "scoop install rancher-desktop", it errors out by saying "It cannot find manifest file". I can install scoop but at a halt when running the above scoop command. Any ideas what the problem could be?

**Comment by jandubois** [comment]
You'll probably have more luck asking the scoop community for help. The Rancher Desktop team does not package the project for scoop; it is done by the scoop project itself. The error you quote seems to indicate that their package is broken.

**Reply by ghost** [reply]
Thank you for responding. ill ask the scoop community
